docs/src/rules/class-methods-use-this.md

```diff
@@ -6,55 +6,91 @@ further_reading:
 - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static
 ---
 
+[Classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes) are often used to encapsulate reusable logic, especially stateful logic, into an object, where each instance's state is accessed via `this`. When an API is written with an instance method, it signals to consumers:
 
-If a class method does not use `this`, it can *sometimes* be made into a static function. If you do convert the method into a static function, instances of the class that call that particular method have to be converted to a static call as well (`MyClass.callStaticMethod()`)
+- The method's outcome is related to the object on which it's invoked, including possibly its state.
 
-It's possible to have a class method which doesn't use `this`, such as:
+	```js
+	const array1 = [1, 2, 3];
+	const array2 = [4, 5, 6];
 
-```js
-class A {
-    constructor() {
-        this.a = "hi";
-    }
+    // Using the `includes()` method on different objects gives different results:
+	array1.includes(1); // true
+	array2.includes(1); // false
 
-    print() {
-        console.log(this.a);
-    }
+	// Modifying the state of an object may change the outcome of its instance methods:
+	array2.push(1);
+	array2.includes(1); // true
+	```
+
+- The method doesn't make sense to be used without an associated object. (For example, it doesn't make sense to call `Array#includes()` without an array to operate on.)
 
+If a class instance method does not use `this`, that normally means that it does not access any instance state and therefore doesn't need to be a method.
+Therefore, it can *sometimes* be refactored into an [ordinary function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions) or a [static method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static), which may better communicate intent to users of the API.
+
+It's possible to have a class method which doesn't use `this`, such as:
```

Comment:
[Flow] The preceding paragraph says _"If a class method does not use `this`, ..."_. That assumes the reader understands such a thing is possible. But this paragraph is what first introduces the concept of a method not including `this`!

Proposal: move this _"It's possible..."_ sentence & its `class Person` code block to before the _"If a class instance..."_ paragraph.